### PR TITLE
Silence warning in FE_Enriched

### PR DESCRIPTION
**Pull request by masterleinad**

### CODE REVIEW

File: source/fe/fe_enriched.cc [hunk 1]
```diff
@@ -188,6 +188,7 @@ FE_Enriched<dim,spacedim>::FE_Enriched (const std::vector< const FiniteElement<
         continue;
 
       const unsigned int base_m  = this->system_to_base_table[system_index].first.second;
+#ifdef DEBUG
```
Comment (bangerth) [inline source/fe/fe_enriched.cc]: Yes, I think that's the better choice.

**Comment by davydden** [comment]
p.s. `clang-apple` does not show those warnings, that's why i have not seen them.

**Comment by masterleinad** [comment]
@davydden Here you go!

**Comment by davydden** [comment]
thanks :+1:

**Pull request merged**